js/tji-donate.js

```diff
@@ -20,14 +20,21 @@ jQuery(document).ready(function($){
 	});
 
 	// Verify form details upon submit
-	$('.tji-donation-submit').click(function() {
-		event.preventDefault();
-		
+	$('#js-donation_form').on("submit", function(e) {
+		e.preventDefault();
+
 		if ($('.selected').length) {
 			var payFee = document.getElementById('tax').checked,
 					donation = parseInt($('.selected').val()),
 					fee = 0,
 					total;
+
+			/* Check for valid donation amount */
+			if (isNaN(donation) || donation == 0) {
```

Comment:
let's try `|| donation <= 0`

Reply:
Hmm, why !isNan? I used "donation == 0" because the pattern matching in the html actually catches negative numbers. Should I go ahead and test here also?

Reply:
ah, I didn't see the validation below since it's not part of the PR. I typo-ed !isNan, and then updated the comment.
So a couple picky things about the custom donation amount validation message -- 
1) could it be cleared if a user clicks another amount button, or if the user refocuses the amount input box.
2) are we intentionally dropping any decimal contributions? so if someone enters 10.50 we drop the 50 by using parseInt?

Reply:
Sure, these are good ideas. I will update the code now.

Reply:
Oh, just saw your comment about rounding confusion. This is happening because you're using parseInt on the inbound value. You can use -- Number.parseFloat(val).toFixed(2);